crowbar_framework/app/models/service_object.rb

```diff
@@ -448,6 +448,12 @@ def node_is_valid_for_role(node, role)
       violates_remotes_constraint?(elements, role)
   end
 
+  # Keeping for backward compatibility only, node_is_valid_for_role
+  # was the original (and confusing) name
+  def node_is_valid_for_role(node, role)
```

Comment:
I am not sure I agree with this. Having both methods ```node_is_valid_for_role``` and ```node_is_invalid_for_role``` present and both of them doing the same thing seems highly confusing to me. And actually I am not able to find a spot out side service_object.rb where ```node_is_valid_for_role``` is currently used.